apolline-flutter/assets/translations/en-GB.json

```diff
@@ -68,6 +68,14 @@
       "name": "Exposure notifications (danger)",
       "description": "Get alerts when current PM values are above danger thresholds."
     },
+    "informationChannel": {
+      "name": "Exposure notifications (inconsistent)",
+      "description": "Gets alerts when current PM values seems inconsistent in order to contextualize your situation."
```

Comment:
"description": "Get alerts when current PM values seem inconsistent in order to contextualize your situation."